src/main/java/com/thealgorithms/backtracking/FloodFill.java

```diff
@@ -79,6 +83,15 @@ public static void printImageArray(int[][] image) {
 		}
 		
 	}
+
+
+	/** Function for JUnit testing **/
+	public static int[][] getFloodFillImage(int image[][], int pixelX, int pixelY, int newColor, int oldColor)
+	{
+		floodFill(image, pixelX, pixelY, newColor, oldColor);
+		return image;
+	}
```

Comment:
Hi, sorry, bit confused.
If I remove getFloodFillImage(), should I remove the JUnit test file also?
I actually created this function so as I could return the resultant array for the test cases I created there.


I was also thinking if it would be a better/clean approach for me to use "image = getFloodFillImage();" instead of floodFill() inside main() function? It would act as a wrapper function. This way, it won't stick out as a separate function, and I could remove the "for JUnit testing" comment from there.

Let me know whichever way works best.

Reply:
You call a floodFill(array) and it mutates array, so you can just assert on that array variable after calling the function